Add Carbon Ads to the footer of each demo example

limonte opened the pull request:
As you maybe noticed, there's the Carbon ads block in the header:

![image](https://user-images.githubusercontent.com/6059356/34640620-e52e38c4-f2fe-11e7-9c68-f0e433e99d4e.png)

[Carbon](https://carbonads.net/) is the invite-only ad network connecting highly qualified audiences with highly relevant services, products, and brands. 

Yesterday I got the offer from them to improve the integration by adding textual ads to the footer of each modal:
 
![image](https://user-images.githubusercontent.com/6059356/34640604-7410a622-f2fe-11e7-958e-b93dbc349dd4.png)

If someone of you guys has the strong disagreement with showing ads this way, please let me know.

By doing this I will be able to say hi via PayPal to all of you more often :) 
We spending our time on producing the high-quality product, and I believe that there should be at least some financial feedback from our good work.

zenflow commented:
@limonte Almost forgot to mention. The new ads aren't working for me in either Chrome (ad blocker disabled) or Firefox (no ad blocker). The `.bsa-cpc` div is being placed where is should be, but they are all empty, including the original copy.

zenflow commented:
Is it working for you?

limonte commented:
> @limonte Almost forgot to mention. The new ads aren't working for me in either Chrome (ad blocker disabled) or Firefox (no ad blocker). The .bsa-cpc div is being placed where is should be, but they are all empty, including the original copy.

@zenflow Hm, strange. Do you still have the same issue? If so, could you please console.log `window._bsa`, it should be something like `{init: ƒ, frequencyCap: ƒ, appendQueryString: ƒ, clearQueue: ƒ, link: ƒ, …}`

zenflow commented:
@limonte Still no

![sweetalert2 github io pr 4](https://user-images.githubusercontent.com/3198597/35551243-39c52086-055c-11e8-9371-2893531b1768.PNG)

toverux commented:
I can see them, no problems :)

I'm generally very agressive towards ads, but I happen to disable my blocker on sites I like and if they do not abuse of ads. Given it's SweetAlert2 and Carbon Ads, I'm not opposed to its introduction.

On the tech side: these are beautifully integrated! Magnificient :smile:
Just thinking: aren't you worried that users could think that the ad could be something we force on their own Swal installation? That recalls me the time when free hosting services were in fact displaying their page with ads in a top bar, and your actual site in an iframe.

I'm unsure this won't be impact-less. Meaning we should display it only on the site at the top (first picture) and in the demo footer modals. Which would strongly diminish ad revenues I presume.

Another note: with an ad blocker, we just have the separator with an empty space after that (see @zenflow's comment), that's horrible. We can't think like an Ad company that likes to think that ad-blockers are a minority. A _lot_ of people use them, therefore a _lot_ of people will see broken modals. We really need to fix that in some way.

limonte commented:
@zenflow I just got the confirmation from BuySellAds that this issue is because of geotargeting:

> - Maybe some geotargeting and there are no ads for the country that user lives in?
> - That’s correct, it happens when there are no ads available for that regions.

You need to relocate in order to see ads on SweetAlert2's page :D

I fixed showing/not-showing of ads footer in https://github.com/sweetalert2/sweetalert2.github.io/commit/310847c558c2e66240022bf3cd03b3b6b12e6b98, you should not see the empty footer anymore.

> I'm generally very agressive towards ads, but I happen to disable my blocker on sites I like and if they do not abuse of ads. Given it's SweetAlert2 and Carbon Ads, I'm not opposed to its introduction.

:tada: 

> On the tech side: these are beautifully integrated! Magnificient
> Just thinking: aren't you worried that users could think that the ad could be something we force on their own Swal installation? That recalls me the time when free hosting services were in fact displaying their page with ads in a top bar, and your actual site in an iframe.

I have the even better idea in my head which will (hopefully) prevent confusions for developers. I will try to separate visually the modal from the footer. 

> Another note: with an ad blocker, we just have the separator with an empty space after that (see @zenflow's comment), that's horrible. We can't think like an Ad company that likes to think that ad-blockers are a minority. A lot of people use them, therefore a lot of people will see broken modals. We really need to fix that in some way.

Fixed in https://github.com/sweetalert2/sweetalert2.github.io/commit/243af3c8f476dca8a46488ad39228f0212e24004 and https://github.com/sweetalert2/sweetalert2.github.io/commit/310847c558c2e66240022bf3cd03b3b6b12e6b98

![peek 2018-01-30 11-26](https://user-images.githubusercontent.com/6059356/35558361-870c8100-05b0-11e8-836d-cd532cb0e224.gif)
